Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/Instance/Instance.cpp

```diff
@@ -228,6 +231,23 @@ namespace AzToolsFramework
             m_entities.clear();
         }
 
+        AZStd::unique_ptr<AZ::Entity> Instance::ReplaceEntity(AZStd::unique_ptr<AZ::Entity>&& entity, EntityAliasView alias)
+        {
+            AZStd::unique_ptr<AZ::Entity> result;
+            auto it = m_entities.find(alias);
```

Comment:
Don't you need to look for the alias in the container entity too?

Reply:
The entities are flattened for spawnables so there's no explicit container entity.

Reply:
There maybe no explicit container entity for the spawnable. But a user provides the alias that corresponds to the container entity of the prefab instance, then it won't be available under 'm_entities'. So, instead of replacing the container entity and returning the original one, you will be returning a nullptr.